src/rt-timeseries/RtTimeseries.tsx

```diff
@@ -69,7 +68,7 @@ const Tooltip: React.FC<{
   return (
     <ChartTooltip>
       <TooltipContents>
-        <TooltipValue>{numeral(value).format("0.0")}</TooltipValue>
+        <TooltipValue>{value}</TooltipValue>
```

Comment:
why

Reply:
Not sure how this happened. No changes should be here.